s64da_benchmark_toolkit/prepare.py

```diff
@@ -74,14 +77,17 @@ def psql_exec_cmd(self, sql):
         return f'psql {self.args.dsn} -c "{sql}"'
 
     def _run_shell_task(self, task, return_output=False):
-        p = Popen(task, cwd=self.benchmark.base_dir, shell=True, executable='/bin/bash',
-                  stdout=PIPE if return_output else None)
-        p.wait()
-        assert p.returncode == 0, 'Shell task did not finish with exit code 0'
-
-        if return_output:
-            stdout, _ = p.communicate()
-            return stdout
+        if not cancelEvent.isSet():
```

Comment:
It actually is `is_set()`.

Reply:
Thanks, I am wondering where I got isSet() from...